taf/tests/test_model/conftest.py

```diff
@@ -0,0 +1,39 @@
+import json
+from pathlib import Path
+
+from taf.tests.conftest import TEST_DATA_PATH
+
+
+from tuf.ngclient._internal import trusted_metadata_set
+from pytest import fixture
+
+
+REPOSITORY_DESCRIPTION_INPUT_DIR = TEST_DATA_PATH / "repository_description_inputs"
+NO_DELEGATIONS_INPUT = REPOSITORY_DESCRIPTION_INPUT_DIR / "no_delegations.json"
+NO_YUBIKEYS_INPUT = REPOSITORY_DESCRIPTION_INPUT_DIR / "no_yubikeys.json"
+WITH_DELEGATIONS_INPUT = REPOSITORY_DESCRIPTION_INPUT_DIR / "with_delegations.json"
+
+
+original_tuf_trusted_metadata_set = trusted_metadata_set.TrustedMetadataSet
```

Comment:
Does `tuf.ngclient` `TrustedMetadataSet` affect these tests as well and is it necessary?  I do remember needing this for updater tests, although I should've wrote down why :(.

Reply:
It wasn't necessary, I must have copied and pasted from another conftest